advanced-cache.php

```diff
@@ -531,6 +531,8 @@ function add_debug_html_to_output( $debug_html ) {
 if ( isset( $_SERVER['QUERY_STRING'] ) ) {
 	parse_str($_SERVER['QUERY_STRING'], $batcache->query);
 	$batcache->query = array_diff_key( $batcache->query, array_flip( $batcache->ignored_query_string_params ) );
+	$batcache->query = array_map( 'urldecode', $batcache->query );
```

Comment:
Why decode?

Reply:
I'm actually not sure about this one in general, the sorting makes sense, let me check if `parse_str()` already decodes.